### ISSUE TITLE
Qemu 7.2 unable to works with virtio

### ISSUE THREAD

**Issue by xrh0905** [AuxXxilium/arc#690]
**Describe the bug**
Using Unraid 6.12.5 with built in Libvirt/Qemu/KVM
Virtio series not functional.

**Screenshots**
If applicable, add screenshots from Arc - Sysinfo menu.

- Least required for complete `find.synology.com` to install DSM
![屏幕截图 2023-12-03 114650](https://github.com/AuxXxilium/arc/assets/29017419/148dba66-61ae-4382-bdde-053adf44d0d9)

- Therory max performance, but only able to find NIC & Disks in `Arc Loader`
![屏幕截图 2023-12-03 120112](https://github.com/AuxXxilium/arc/assets/29017419/60540c2e-b8d9-4b9f-a6d7-70e0c74e4d78)

**Additional context**
Same issues was sent to `wjz304/rr`, but not resolve before `rr` is gone.
Willing to provide any more required details.
Boot log from serial0 attached below.
[boot20231203.log.txt](https://github.com/AuxXxilium/arc/files/13538018/boot20231203.log.txt)

**Comment by xrh0905** [comment]
Hi.Sorry for the late reply due to the timezone difference 
For the first question  - I followed the link I found in the previous issue that was posted by you in #320 . However, while I was testing, there are something that is woth notice.
- qemu XHCI USB : no `synobootX` device present in DSM
- qemu EHCI USB: `synobootX` pesent
- SATA0: `synobootX` present
I believe  that. this is related to the delicate drivers of XHCI isn't being properly loaded as well.
As of the latter question - Disks are connected via `VirtIO SCSI` - They can be found with `ls /dev/sdX` in the Arc loader but not in the junior mode. I forgot to remove the SATA controller but I assume that this wasn't related.
The main problem I'm facing is that all the VirtIO related stuff isn't working in the DSM including `scsi` `net` and `console` which are necessary for optimal performance and guest agent.
Another fact that I had to mention is that with similar settings on my proximal server (SATA0 Arc SCSI0-3 Data Q35/Ovmf  Virtio) I was able to boot the system without issues, DSM recognized the NIC and the disk (13 to 16 since delete  sata controller is hard in PVE) properly and I am hoping to get a similar result with my unraid mainframe.
I've been working on this for a while and with `dmesg|grep virtio` I noticed some weird error that I was never seen before which also presenting the boot log I provided. `lsmod` only gives `virtio`, `virtio_ring` and `virtio_pci` but missing others.
I'm looking forward to your reply on how should I diagnose this further and I'm willing to tweak with advanced settings. Thanks for your patience and precious time and understood my English as I am not a native speaker.
